core/node/consistency_checker/src/lib.rs

```diff
@@ -532,7 +534,10 @@ impl ConsistencyChecker {
 
         while let Err(err) = self.sanity_check_diamond_proxy_addr().await {
             if err.is_transient() {
-                tracing::warn!("Transient error checking diamond proxy contract; will retry after a delay: {err}");
+                tracing::warn!(
+                    "Transient error checking diamond proxy contract; will retry after a delay: {:#}",
+                    anyhow::Error::from(err)
+                );
```

Comment:
There is certainly a better way to output the chain of error sources (e.g., it doesn't need to consume the error), but I'm not sure where to place the corresponding helper; converting an error to `anyhow::Error` looks like an OK-ish alternative.